.yarnrc

```diff
@@ -1,2 +0,0 @@
-save-prefix ""
```

Comment:
how do we deal with save exact version behavior?

Reply:
We can change it manually inside package.json